app/assets/stylesheets/base/reset.scss

```diff
@@ -67,7 +68,7 @@ pre,
 code,
 kbd,
 samp {
-  font-family: monospace, monospace;
+  font-family: var(--ff-accent);
```

Comment:
Is this `nonospace`?

Reply:
Yes, it's a monospace font. I think this variable name is a bit misleading - I'll rename it to `--ff-monospace` in a separate PR.